### DISCUSSION TITLE
Q3| Technical Manager Proposal

### DISCUSSION THREAD

**Discussion by furoxr** [ringdao-community-guild/discussion#28] [General]
## Introduction

I’m seeking a monthly position in Q3. After months of work in DCDAO, I have made a upgrade for Bridger to support on-demand header relay, I have helped build DCDAO such as the Profiles NFT application workflow, and I have made some integratioin such as Safe contracts, Multicall and snapshots. This quarter I will focus on these topics:

1. Development and maintenance

2. Technical support and education

3. Promote integration with other entities

### Development and maintenance:

I plan to continue maintaining resources like repositories and tools, such as profiles, awesome-darwinia, slack-discord message bot and so on. And I'm available to contribute towards developing new tools or functionalities as needed.

### Technical support and education:

I plan to write some technical articles to help people use Darwinia Network and expand the influence. And I plan to make some data analysis about on-chain data such as staking, RING etc.

### Promote integration with other entities:

I plan to reach out to more service providers to promote the integration with Darwinia, which is public good and will benefit Darwinia Network users, such as Safe Gnosis, Debank, Coingecko and so on.

What's more, I will post a report every month to elaborate the contribution and the work plan for next month.

## Budget

The funding I’m seeking is $3200 every month(Both USDT and RING are acceptable), and my ethereum address is: 0xa6B5EC845C8446A350a50bb58363c241ab7990b5

**Comment by hackfisher** [comment]
Can you kindly add and provide monthly reports in your deliveries of the proposal?

**Reply by furoxr** [reply]
THX~ I have updated
